src/winforms/toga_winforms/window.py

```diff
@@ -114,13 +115,12 @@ def vertical_shift(self):
         return result
 
     def set_content(self, widget):
-        for control in self.native.Controls:
-            self.native.Controls.Remove(control)
-
         if self.toolbar_native:
             self.native.Controls.Add(self.toolbar_native)
```

Comment:
Re-adding the toolbar without removing it first should be a no-op, but I'll test this more thoroughly once we've confirmed this is the right approach.